core/client_config.py

```diff
@@ -57,22 +57,7 @@ def get_setup_details(self, target):
         return setup_details
 
     @staticmethod
-    def run_cmd(cmd):
```

Comment:
This function is being used in the same class.  There was a reason to have it in the same class if I remember correctly.

Reply:
@seagate-sarang-sawant not 100% sure what was the reason.  
And this client config flow is not applicable currently as we have updated building endpoint in code base. So this configuration will need some rework. 
This "run_cmd" was being used to remove "/etc/hosts" file and generate new one and execute make command for configuration.

Reply:
This code path is used in case the corbot runs as kafka consumer for automated deployment.  Keep the changes as of now unless we revisit the execution framework Gliese.

Reply:
Okay, changed back